proposals/hackygolucky-EXPECTATIONS/EXPECTATIONS.md

```diff
@@ -0,0 +1,108 @@
+## Expectations
+
+With the intent to merge the Node.js Foundation and JS Foundation being announced, those interested in the continued support of Node.js and JS-related projects are working through expectations and structure for what the potential new foundation could look like.
+
+This document is for the initial phase of work for the Bootstrap team to draft requirements that overlap with scope of the Board in order to provide an outline and structure for the foundation attuned to the needs of the projects and ecosystem stakeholders who have participated.
+
+This document is organized by two key areas: concerns that must be included in the foundation bylaws, and those that are communicated expectations that aren't required to be in our legal documents to proceed but are vital to managing expectations of project members and ecosystem stakeholders.
+
+### Bylaws requirements
+- Communicated scope for the foundation and board 
+- Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement 
+- The foundation staff, executive director, and board should follow a leadership style that prioritizes and promotes the well-being of the project and those who prioritize and promote a healthy project and ecosystem. 
+- Project representation on the Board 
```

Comment:
I'd like us to add

+ - Provide a place where the broader and longer-term issues important to the success of the overall JavaScript ecosystem can discussed, championed and supported.

I think this should show up on the structure charts as some sort of box were we have an expectation that there will be a group of people within the foundation working on these issues in a way that is formally supported by the project.  This might be the CPC, but I'm not sure that will be the case as if we have both a CPC and C3 I don't think that mandate is necessarily limited to one of the 2.

Reply:
+1